tools/StaticAnalysis/Exceptions/SignatureIssues.csv

```diff
@@ -1547,4 +1547,10 @@
 "Microsoft.Azure.Commands.DataLakeAnalytics.dll","Microsoft.Azure.Commands.DataLakeAnalytics.SetAzureDataLakeAnalyticsAccount","Set-AzureRmDataLakeAnalyticsAccount","1","8410","Parameter MaxAnalyticsUnits of cmdlet Set-AzureRmDataLakeAnalyticsAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.DataLakeAnalytics.dll","Microsoft.Azure.Commands.DataLakeAnalytics.SubmitAzureDataLakeAnalyticsJob","Submit-AzureRmDataLakeAnalyticsJob","1","8410","Parameter AnalyticsUnits of cmdlet Submit-AzureRmDataLakeAnalyticsJob does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.DataLakeAnalytics.dll","Microsoft.Azure.Commands.DataLakeAnalytics.UpdateAzureDataLakeAnalyticsComputePolicy","Update-AzureRmDataLakeAnalyticsComputePolicy","1","8410","Parameter MaxAnalyticsUnitsPerJob of cmdlet Update-AzureRmDataLakeAnalyticsComputePolicy does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
-"Microsoft.Azure.Commands.MachineLearningCompute.dll","Microsoft.Azure.Commands.MachineLearningCompute.Cmdlets.RemoveAzureRmMlOpCluster","Remove-AzureRmMlOpCluster","1","8410","Parameter IncludeAllResources of cmdlet Remove-AzureRmMlOpCluster does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
\ No newline at end of file
+"Microsoft.Azure.Commands.MachineLearningCompute.dll","Microsoft.Azure.Commands.MachineLearningCompute.Cmdlets.RemoveAzureRmMlOpCluster","Remove-AzureRmMlOpCluster","1","8410","Parameter IncludeAllResources of cmdlet Remove-AzureRmMlOpCluster does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.Commands.Kubernetes.dll","Microsoft.Azure.Commands.Kubernetes.GetAzureRmKubernetes","Get-AzureRmKubernetes","1","8400","Get-AzureRmKubernetes uses the noun 'AzureRmKubernetes', which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."
```

Comment:
I think we should probably add 'Kebernetes' as one of our exceptions to the plurals rule, but this will do for now

Reply:
'Kubernetes' :)